e2e-tests/playwright/e2e/github-happy-path.spec.ts

```diff
@@ -27,6 +27,9 @@ test.describe.serial("GitHub Happy path", () => {
     backstageShowcase = new BackstageShowcase(page);
     await common.loginAsGithubUser();
   });
+  test.beforeEach(
+    async () => await new Common(page).checkAndClickOnGHloginPopup(),
+  );
```

Comment:
@nilgaar I don't know if this will work or not. In the case of this [test](https://github.com/PatAKnight/backstage-showcase/blob/51b179309d00dfa2117ccdece3141ea915e14968/e2e-tests/playwright/e2e/github-happy-path.spec.ts#L96), we navigate to the `Issues` tab which sometimes will trigger the GitHub login popup. 

I think we may need a more targeted approach with some of these tests that trigger login popups, WDYT?